docs/source/guides.md

```diff
@@ -0,0 +1,42 @@
+# How-to Guides
+
+This page contains guides of how to perform common actions using the library.
+
+## Discover devices
+
+```{eval-rst}
+.. automodule:: kasa.discover
+```
+
+## Connect without discovery
+
+```{eval-rst}
+.. automodule:: kasa.deviceconfig
+```
+
+## Get Energy Consumption and Usage Statistics
+
+```{note}
+    In order to use the helper methods to calculate the statistics correctly, your devices need to have correct time set.
+    The devices use NTP and public servers from `NTP Pool Project <https://www.ntppool.org/>`_ to synchronize their time.
+```
```

Comment:
```suggestion
```{note}
In order to use the helper methods to calculate the statistics correctly, your devices need to have correct time set.
The devices use NTP and public servers from `NTP Pool Project <https://www.ntppool.org/>`_ to synchronize their time.
```
```
This is currently rendered as a code block in the generated docs, but I think de-indenting should fix that.

Reply:
Done.  Enabled [colon_fence](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html#admonitions) myst extension in the process as per advice.

Reply:
Nice find, colon fences look much nicer, too :+1: